bindings/python/src/model/representation/core/brep.h

```diff
@@ -101,8 +95,7 @@ namespace geode
             .def( "nb_internal_surface_corners",
                 ( index_t( BRep::* )( const Surface3D& ) const )
                     & BRep::nb_internal_corners )
-            .def(
-                "internal_surface_corners",
```

Comment:
what about internal corners inside lines?
Also, thanks to the dimensions, we guess that the internal object is the corner but might be more obvious with internal_corners_inside_surface?

Reply:
An internal corner in a ligne is just defining a boundary of two lines. It's our convention. What do you think?

Reply:
I am ok if we can have make collections of lines. I didn't check.